osgeo_importer/views.py

```diff
@@ -79,40 +86,50 @@ class FileAddView(FormView, ImportHelper, JSONResponseMixin):
     json = False
 
     def form_valid(self, form):
-        upload = UploadedData(user=self.request.user)
+        upload = UploadedData.objects.create(user=self.request.user)
         upload.save()
 
         # Create Upload Directory based on Upload PK
-        outpath = os.path.join('/uploads',str(upload.pk))
-        outdir = os.path.join(settings.MEDIA_ROOT,outpath)
+        outpath = os.path.join('osgeo_importer_uploads',str(upload.pk))
+        outdir = os.path.join(FileSystemStorage().location,outpath)
+        if not os.path.exists(outdir):
+            os.makedirs(outdir)
+        logger.debug("%s",outdir)
+        logger.debug("cleaned_data: %s",form.cleaned_data)
```

Comment:
this log message could be merged with the previous one, if one or both aren't just to be removed